composer.json

```diff
@@ -23,31 +23,30 @@
     "require": {
         "php": "^7.3 || ^8.0",
         "ext-json": "*",
-        "doctrine/doctrine-bundle": "^1.12 || ^2.0",
-        "doctrine/persistence": "^1.3 || ^2.0",
+        "doctrine/doctrine-bundle": "^2.0",
+        "doctrine/persistence": "^2.0",
         "laminas/laminas-diagnostics": "^1.6",
         "sonata-project/datagrid-bundle": "^3.0",
         "sonata-project/doctrine-extensions": "^1.10.1",
         "sonata-project/form-extensions": "^1.4",
-        "symfony/config": "^4.4 || ^5.2",
-        "symfony/console": "^4.4 || ^5.2",
-        "symfony/dependency-injection": "^4.4 || ^5.2",
-        "symfony/doctrine-bridge": "^4.4 || ^5.2",
-        "symfony/event-dispatcher": "^4.4 || ^5.2",
+        "symfony/config": "^4.4 || ^5.3",
+        "symfony/console": "^4.4 || ^5.3",
+        "symfony/dependency-injection": "^4.4 || ^5.3",
+        "symfony/doctrine-bridge": "^4.4 || ^5.3",
+        "symfony/event-dispatcher": "^4.4 || ^5.3",
         "symfony/event-dispatcher-contracts": "^1.1 || ^2.0",
-        "symfony/form": "^4.4 || ^5.2",
-        "symfony/http-foundation": "^4.4 || ^5.2",
-        "symfony/http-kernel": "^4.4 || ^5.2",
-        "symfony/security-core": "^4.4 || ^5.2"
+        "symfony/form": "^4.4 || ^5.3",
+        "symfony/http-foundation": "^4.4 || ^5.3",
+        "symfony/http-kernel": "^4.4 || ^5.3",
+        "symfony/security-core": "^4.4 || ^5.3"
     },
     "conflict": {
         "friendsofsymfony/rest-bundle": "<2.3",
         "jms/serializer": "<0.13",
-        "liip/monitor-bundle": "<1.0",
+        "liip/monitor-bundle": "<2.6",
```

Comment:
do we really need the bump of the conflict ?

Reply:
Well we are not testing with with liip monitor bundle 1.0, so we don't even know if it works or not. (on next major imo we should bump most of the versions to not support more than needed)